src/Microsoft.ML.AutoML/CodeGen/fast_tree_search_space.json

```diff
@@ -75,6 +75,11 @@
     {
       "name": "ExampleWeightColumnName",
       "type": "string"
+    },
+    {
+      "name": "DiskTranspose",
```

Comment:
This option allows user to overwrite [`DiskTranspose`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.trainers.fasttree.treeoptions.disktranspose?view=ml-dotnet) when necessary, which avoids OOM when dataset is large